src/components/CreatePost.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <header>
+    <NavBar />
+  </header>
+
+  <div class="container">
+    <div class="row">
+      <div class="col-12 text-center">
+        <h2 class="pt-3">Create Post</h2>
+      </div>
+    </div>
+    <div class="row">
+      <div class="col-3"></div>
+      <div class="col-6">
+        <form @submit.prevent="savePost">
+          <div class="form-group">
+            <label>ImageURL:</label>
+            <input
+              type="text"
+              v-model="post.image"
+              class="form-control"
+              required
+            />
+          </div>
+          <div class="form-group">
+            <label>Tags:</label>
+            <select v-model="post.selectedTags" multiple class="form-control">
```

Comment:
This select component is not the best as someone less technical won't know how to select multiple items without trying hard.

You can build your own tags select with chips or use any open source package like [Vue Form Multiselect](https://github.com/vueform/multiselect). Use multi-select with Tags to achieve the desired result.